lighthouse-core/report/html/renderer/util.js

```diff
@@ -496,7 +496,9 @@ Util.i18n = null;
  */
 Util.UIStrings = {
   /** Disclaimer shown to users below the metric values (First Contentful Paint, Time to Interactive, etc) to warn them that the numbers they see will likely change slightly the next time they run Lighthouse. */
-  varianceDisclaimer: 'Values are estimated and may vary. The [performance score is calculated](https://web.dev/performance-scoring/) directly from these metrics.',
+  varianceDisclaimer: 'Values are estimated and may vary. The [performance score is calculated](https://web.dev/performance-scoring/) directly from these metrics. ',
```

Comment:
```suggestion
  varianceDisclaimer: 'Values are estimated and may vary. The [performance score is calculated](https://web.dev/performance-scoring/) directly from these metrics.',
```

Reply:
if we need spacing between these let's do CSS rather than affect the translations? I'm nearly certain translators would not preserve this space as critical to layout :)

Reply:
Could we stick these strings together? Slightly more awkward to deal with, but "See calculator" could default link to `https://googlechrome.github.io/lighthouse/scorecalc/` and then `performance-category-renderer` could enhance that by reaching into the string span, finding the calculator link and replacing it with the parameterized link

Reply:
did with css spacing

@brendankenny part 2 of this (mentioned in #10726) is to just reuse the same 'perf score is calculated' link to do all of it.

Reply:
> @brendankenny part 2 of this (mentioned in #10726) is to just reuse the same 'perf score is calculated' link to do all of it.

FWIW we shouldn't really concat these i18n-wise, but I guess everyone will be getting English strings until they're translated anyways :)

Reply:
> FWIW we shouldn't really concat these i18n-wise, but I guess everyone will be getting English strings until they're translated anyways :)

actually I still don't understand how that idea precludes doing my idea or requires the current impl in the meantime (since the base string was just changed in #10725 and won't be translated anyways), but ok :)

Reply:
>  (since the base string was just changed in #10725 and won't be translated anyways), but ok :)

that change is a noop to the TC pipeline, in fact. :)